libs/core/src/lib/date-picker/date-picker.component.html

```diff
@@ -9,27 +9,29 @@
     <fd-popover-control>
         <fd-input-group [compact]="compact" [state]="state" [disabled]="disabled">
             <input
+                type="text"
                 class="fd-input"
                 fd-input-group-input
-                type="text"
+                [compact]="compact"
+                [disabled]="disabled"
+                [placeholder]="placeholder"
+                [class.is-error]="isInvalidDateInput && useValidation"
```

Comment:
this is a nice change, I like it :D

Reply:
Look guys! I'm not crazy! Other people also find this grouping useful 😆

Reply:
One thing though, this will be executed on each change detection cycle, because angular doesn't have concept of computed properties.
What I really like is removing of ngClass/ngStyle usage, which are slow and bring the whole module with it